src/components/DaoCard.tsx

```diff
@@ -17,6 +17,11 @@ interface DaoCardProps {
 
 const DaoCard = (props: DaoCardProps) => {
   const [currentWalletAccount] = useElioStore((s) => [s.currentWalletAccount]);
+
+  const isOwner =
```

Comment:
TODO: change to hasOwnedTokens

Reply:
You make make hasOwnedTokens = currentWalletAccount && !daoTokenBalance.isZero() 

And fetch daoTokenBalance in ExploreDao component or something.